src/core/net/ip6_mpl.cpp

```diff
@@ -334,6 +334,27 @@ uint8_t Mpl::DetermineMaxRetransmissions(void) const
     return maxRetx;
 }
 
+bool Mpl::RemoveMatchedMessage(Message &aMessage)
+{
+    bool     match = false;
+    uint16_t len   = aMessage.GetLength();
+    VerifyOrExit(len > 0);
```

Comment:
Newline after var declaration (per [style guide](https://github.com/openthread/openthread/blob/a5c17b77635bb43d02d4dec96fbf4b7eeb43be06/STYLE_GUIDE.md#white-space)).

```suggestion
    uint16_t len   = aMessage.GetLength();
    
    VerifyOrExit(len > 0);
```

Reply:
thanks, fixed it